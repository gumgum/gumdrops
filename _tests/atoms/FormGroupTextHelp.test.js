import renderer from 'react-test-renderer';
import React from 'react';
import FormGroupTextHelp from '../../components/atoms/FormGroupTextHelp';

test('Expect <FormGroupTextHelp> to render properly', () => {
    const defaultProps = {
        text: 'Oh no, errors!',
        className: 'much-error'
    };
    const tree = renderer.create(<FormGroupTextHelp {...defaultProps} />).toJSON();
    expect(tree).toMatchSnapshot();
});
