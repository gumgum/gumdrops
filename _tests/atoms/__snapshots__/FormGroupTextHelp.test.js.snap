// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect <FormGroupTextHelp> to render properly 1`] = `
<small
  className="gds-form-group__text-help much-error"
>
  Oh no, errors!
</small>
`;
